src/Microsoft.AspNet.Mvc.ViewFeatures/ViewResult.cs

```diff
@@ -79,6 +79,7 @@ public class ViewResult : ActionResult
                             actionContext = context,
                             result = this,
                             viewName = viewName,
+                            viewEgineType = viewEngine.GetType(),
```

Comment:
This is almost certainly not going to do what you want. What is this needed for? Why didn't it come up in our requirements scrub that we did earlier in beta8.